rosenpass/src/cli.rs

```diff
@@ -176,7 +183,8 @@ pub enum CliCommand {
         ///
         /// OUTFILE := outfile <PATH>
         ///
-        /// WG := wireguard <WIREGUARD_DEV> <WIREGUARD_PEER> [WIREGUARD_EXTRA_ARGS]...
+        /// WG := wireguard <WIREGUARD_DEV> <WIREGUARD_PEER>
+        /// [WIREGUARD_EXTRA_ARGS]...
```

Comment:
Some comment than above: This change wraps command-line arguments. Does this end up in an alright format when rendered in various formats? Independent of the wrapping, does this need backticks?

Reply:
Same answer as above https://github.com/rosenpass/rosenpass/pull/486#discussion_r1838207339 :)